bootcamp/tutorials/integration/llamaindex/llamaindex_milvus_full_text_search.ipynb

```diff
@@ -0,0 +1,437 @@
+{
```

Comment:
<p>The&nbsp;<code>MilvusVectorStore</code>&nbsp;abstraction in LlamaIndex has supported full-text search. By integrating a built-in function as the&nbsp;<code>sparse_embedding_function</code>, it can search text using the BM25 scoring.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/milvus-io/bootcamp/pull/1516/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>